src/public/js/EphemeralMessage/index.js

```diff
@@ -1,15 +1,21 @@
 import store from '@js/store';
 import {renderEphemeralDetails} from '@js/EphemeralMessageRenderer';
 import isPlainObject from 'lodash/isPlainObject';
-import {addSystemMessage} from '@js/Togethernet/systemMessage';
+import {addSystemConfirmMessage} from '@js/Togethernet/systemMessage';
+import {addSystemNotifyMessage} from '@js/Togethernet/systemMessage';
+import {systemConfirmMsg_consentToArchive} from '@js/constants.js';
+import {systemConfirmMsg_initiateConsentToArchiveProcess} from '@js/constants.js';
+import {systemNotifyMsg_blockConsentToArchive} from '@js/constants.js';
```

Comment:
same comment re: snake case and camelcase

Reply:
Resolved.